modules/coreI18n/src/main/key.scala

```diff
@@ -968,6 +968,10 @@ object I18nKey:
     val `sanPromotesTo`: I18nKey = "nvui:sanPromotesTo"
     val `sanShortCastling`: I18nKey = "nvui:sanShortCastling"
     val `sanLongCastling`: I18nKey = "nvui:sanLongCastling"
+    val `sanDroppedOn`: I18nKey = "nvui:sanDroppedOn"
+    val `invalidMove`: I18nKey = "nvui:invalidMove"
+    val `premoveCancelled`: I18nKey = "nvui:premoveCancelled"
+    val `premoveRecorded`: I18nKey = "nvui:premoveRecorded"
```

Comment:
these are supposed to be generated from the nvui.xml file, but I don't see the 4 messages there?

Reply:
https://github.com/yafred/lila/blob/a78ae99f6ee17a3b50810a50d1da9a80adce78df/translation/source/nvui.xml#L71

Reply:
FYI I just let my French nvui.xml go with the next commit ... I don't think it's a problem